tests/utils/test_codegen.py

```diff
@@ -32,6 +32,114 @@ def test_compare_attributes(self):
         self.assertEqual(1, ClassUtils.INCLUDES_SOME)
         self.assertEqual(2, ClassUtils.INCLUDES_ALL)
 
+    @mock.patch.object(ClassUtils, "sanitize_sequential")
+    @mock.patch.object(ClassUtils, "sanitize_restrictions")
+    @mock.patch.object(ClassUtils, "sanitize_attribute")
+    def test_sanitize_attributes(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=5244803)